src/common/contracts/ao-process.ts

```diff
@@ -44,9 +44,11 @@ export class AOProcess implements AOContract {
   async read<K>({
     tags,
     retries = 3,
+    Owner,
   }: {
     tags?: Array<{ name: string; value: string }>;
     retries?: number;
+    Owner?: string;
```

Comment:
wondering if we should do something like Arguments<this.ao.dryrun>[0] and pass those all thru.

In any case, I suspect what you want here may be `From`, not `Owner` - right?

Reply:
yeah but the type on dryrun only has `Owner` and it seems to do what we need. previously we had no way of indicating the From address on a read action
<img width="576" alt="image" src="https://github.com/user-attachments/assets/4cb6c17d-4934-46d8-ad26-435c2a793dec" />

Reply:
exposed as fromAddress in our API fwiw: ee4a024

Reply:
Interesting - it must be adding it on its own if no From param is provided. 

This may have issues if attempting to do a dry run as an ANT (because of authorities) - but this is probably fine for now.